noxfile.py

```diff
@@ -53,7 +53,10 @@ def tests(session):
     session.log("Starting tests. See the logs to analyze.")
     for file in os.listdir("tests"):
         if file.endswith(".diddi"):
-            session.run("diddiparser2", f"tests/{file}")
+            if session.posargs:
+                session.run("diddiparser2", f"tests/{file}", *session.posargs)

```

Comment:
It is now possible to give posargs to the DiddiScript command when running `nox -s tests`. For example, you pass `--verbose` by running `nox -s tests -- --verbose` :wink:.